src/components/existing-request/existing-request-display.vue

```diff
@@ -204,13 +206,6 @@
             v-if="showNrNotApprovedGrayBox"
             :nrNum="nr.nrNum"
           />
-
-          <!-- incorporate button -->
-          <div class="mt-5 text-center" v-if="showIncorporateButton">
-            <v-btn id="INCORPORATE-btn" @click="handleButtonClick(NrAction.INCORPORATE)">
```

Comment:
This button is now in the `NRApprovedGrayBoy` above. I'm following the UXPin. It has similar behavior as the `Register Your Business` button